modules/devices/languages/en.json

```diff
@@ -0,0 +1,54 @@
+{
+    "DEVICES": {
+        "MODULE_DETAILS": {
+            "SINGULAR_NAME": "Device",
+            "PLURAL_NAME": "Devices"
+        },
+        "FIELDS": {
```

Comment:
Add Japanese (ja.json), Korean (ko.json), and Simplified Chinese (zh_cn.json) JSONs for `devices` module